labcontrol/gui/templates/plate.html

```diff
@@ -206,15 +206,19 @@ <h4>Plate map</h4>
           <section id='plate-map-legend-indeterminate' class='legend-entry'>
             <div id='plate-map-legend-indeterminate-box' class='well-indeterminate legend-patch'></div>
             <div id='plate-map-legend-indeterminate-text' class='legend-text'>
-             <p> Specimen ID matches more than one member of the attached studies</p>
+             <p> Specimen ID matches more than one member of the attached
+             studies; if left unresolved, will show up as "not a member of the
+             attached studies" when reloading this plate</p>
             </div>
           </section>
           <!-- Hidden section added to populate a full row; otherwise, Well
               comments will not be left-justified. -->
```

Comment:
I hadn't noticed the duplicate IDs, but I agree that that is egregious. Just pushed a commit that should fix that particular wart, at least.

It looks like duplicating the text is needed (at least with the way the HTML/CSS is written currently) because truncating the hidden text messes with the layout of the next section ("Well Comments")—

<img width="1287" alt="Screen Shot 2019-09-09 at 7 21 20 PM" src="https://user-images.githubusercontent.com/4177727/64579342-66086900-d337-11e9-94f7-6b6fadb78704.png">

For comparison, here's a screenshot with the duplicated text solution in use—

<img width="1219" alt="Screen Shot 2019-09-09 at 7 24 53 PM" src="https://user-images.githubusercontent.com/4177727/64579377-8801eb80-d337-11e9-81dd-78d4d0ab18d8.png">

I changed the hidden divs to be visible for these screenshots, but the effects shown here persist regardless of visibility.

Anyway! Yeah, entirely agreed that the need for this hidden section at all is a hack (and I hereby take responsibility for that, since I approved this code when it was merged in :). Removing the duplicate IDs makes it more tolerable, but it'd definitely be a good idea to adjust this to make the bottom row expand without needing a hidden section. I just created #590 to address this (can try to fix that in this PR if you'd like, but I don't think it's super urgent).